katello/katello.spec

```diff
@@ -28,15 +28,57 @@ BuildRequires: util-linux
 
 Requires: %{name}-common = %{version}-%{release}
 
+%if 0%{?rhel} == 6
+Requires: redhat-logos >= 60.0.14
+%endif
+
+Requires(post): chkconfig
+Requires(postun): initscripts coreutils sed
+Requires(pre): shadow-utils
+Requires(preun): chkconfig
+Requires(preun): initscripts
```

Comment:
Glad you asked that as I wondered the same thing myself -- I don't see any post or pre service related items given we no longer worry about deploying our own service or tasking anymore.